src/Engine/ProtoCore/RuntimeCore.cs

```diff
@@ -64,7 +64,7 @@ public void Reset()
     /// </summary>
     public class RuntimeCore
     {
-        public RuntimeCore(Heap heap)
+        public RuntimeCore(Heap heap, Options options = null)
```

Comment:
the message log test that fails requires the options to be present.
The root cause has to do with the LiveRunner being intialized multiple times when running test cases